benchmarks/functions/__init__.py

```diff
@@ -27,11 +37,17 @@ def setup_benchmark(self, function, inputs, grad_outputs=None):
         self.function = function
 
         # Prepare for forward.
-        self.forward_inputs = (
-            [None if x is None else chainer.Variable(x) for x in inputs])
+
+        self.forward_inputs = ([self._convert_to_variable(x) for x in inputs])
 
         # Prepare for backward.
-        outputs = chainer.functions.identity(self.forward())
+        ret = self.forward()
+
+        if isinstance(ret, tuple):
```

Comment:
How about this for simplicity?

```py
if not isinstance(ret, (list, tuple)):
    ret = (ret,)
outputs = chainer.functions.identity(*ret)
```

Reply:
Simplified these lines by using _normalize_outputs().